process/customer-success.md

```diff
@@ -0,0 +1,409 @@
+
+* Number of users visiting Marketing sites
+* Number of users visiting the openebs - code and docs sites
+* Number of docker pull, cluster call home metrics
+* Number of users visiting the MayaOnline 
+* Number of users registering with MayaOnline
+* Number of users connecting the clusters to MayaOnline
+* Average time of the clusters in MayaOnline by the users. 
+* Number of users requesting for Evaluation
+* Number of users subscribing 
+* Number of users renewing the subscriptions
+
+## Feel the Pulse of Customers. Happiness Index. 
+
+*Are our customers satisfied? What can we improve?*
+
+Survey your clients and get a reading on how satisfied they are. Hearing directly from your customers can help paint a picture that tickets alone may not be able to provide. Reach out to the customers with different survey depending on their engagement. 
+
+* Idle Customers - who have signed up but have not raised a ticket or connected their clusters 
+* Raised Tickets have been resolved. Reach out with a CAST survey to check for:
+    * Satisfaction in terms of handling the issue
+    * Suggestions for improvements in getting better experience. Can the docs, KB or the products be improved. 
+    * Understand the use case at a broader sense. 
+* Posting queries about the product or upgrade questions. Reach out with HXC survey checking with customers on:
+    * Finding out how much value OpenEBS is adding to their DevOps tasks
+    * What Problems it is helping solve
+    * Check if they will become a reference user
+    * Find out the pain points with OpenEBS
+* Stopped being active. Reach out with Feedback survey to identify the reasons for moving away from OpenEBS.
+
+## Ticket Analysis
+
+Analyse the ZenDesk tickets, FullStory recording to identify:
+
+* Documentation Improvements
+* Common Product Issues - that are hindering adoption
+* Time of Day or dates when the volumes of the tickets is more. Check if more team members need to be onboarded with Customer Advocacy. 
+* Measure the SLA for different issues
+* Follow up on the issues that have been resolved.
+
+Chart out the plan and track the progress and measure the improvements. 
+
+## Motivated Customer Success Team
+
+*How are the individuals on the team performing and feeling? *
+
+Conduct ESAT survey with the employees - that are both involved and not yet involved. Check on what hinders them from being effective in Customer Advocacy.
+
+# Appendix
+
+## Escalation Matrix
+
+The customers can use the following Escalation Matrix:
+
+* Report an issue via Slack or MayaOnline or via Email to ( [support@mayadata.io](mailto:support@mayadata.io) )
+* Send an email to Support Leads in case there is no response to ( [support-leads@mayadata.io](mailto:support-leads@mayadata.io) ). The ZenDesk will be automatically setup SLA triggers for initial response. The customers are also provided with US 1800 Phone.
+
+## Support Shift Schedule
+
```

Comment:
How about the staffing part should be part of this document, or link to a new document (Leaves, holidays in a week) To meet 24/7